hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/string/LikeFunctionIntegrationTest.java

```diff
@@ -37,132 +43,149 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-@RunWith(HazelcastParallelClassRunner.class)
+@RunWith(Parameterized.class)
+@Parameterized.UseParametersRunnerFactory(HazelcastParallelParametersRunnerFactory.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class LikeFunctionIntegrationTest extends ExpressionTestSupport {
+
+    private static final ConstantExpression<?> CONST_1 = ConstantExpression.create("1", QueryDataType.VARCHAR);
+    private static final ConstantExpression<?> CONST_2 = ConstantExpression.create("2", QueryDataType.VARCHAR);
+    private static final ConstantExpression<?> CONST_3 = ConstantExpression.create("3", QueryDataType.VARCHAR);
+    private static final ConstantExpression<?> CONST_OTHER = ConstantExpression.create("100", QueryDataType.VARCHAR);
+
+    @Parameterized.Parameter
+    public boolean negated;
+
+    @Parameterized.Parameters(name = "mode:{0}")
+    public static Collection<Object[]> parameters() {
+        return Arrays.asList(new Object[][]{ { false }, { true } });
```

Comment:
If you have just one parameter, you don't need to use a nested collection:
```suggestion
    public static List<Boolean> parameters() {
        return Arrays.asList(false, true);
```

Reply:
Fixed as suggested.